### ISSUE TITLE
Error when using jsTree v3.3.6+ with jQuery 3.4.1

### ISSUE THREAD

**Issue by Rikijs** [vakata/jstree#2253]
Hello!

Tree works fine while using `jstree v3.3.6` with `jquery v3.3.1`

But i am encountering an error when using `jstree v3.3.6` (or newer) with `jquery v3.4.1`

**ERROR**

```
jQuery.Deferred exception: $project_tree.jstree is not a function ./assets/js/project_tree_show.js/<@http://127.0.0.1:8000/build/tree_show.26f50966.js:30:17
mightThrow@http://127.0.0.1:8000/build/vendors~app~blueimp~clipboard~delete_folder~page_auto_refresh~tree_delete~tree_restore_1~tree_restor~ed1dc121.9d333f1e.js:3566:29
resolve/</process<@http://127.0.0.1:8000/build/vendors~app~blueimp~clipboard~delete_folder~page_auto_refresh~tree_delete~tree_restore_1~tree_restor~ed1dc121.9d333f1e.js:3634:12
 undefined jquery.js:3841
TypeError: $project_tree.jstree is not a function
```

**MY CODE**

```
use strict';

// loads the jquery package from node_modules
import $ from 'jquery';

// add jstree
import jstree from 'jstree';
import 'jstree/dist/themes/default/style.min.css';

$(function()
{
    let $project_tree, $search_input, $search_reset, $project_tree_loader;

    $project_tree = $('#file-tree');
    $project_tree_loader = $('#js-file-tree-loader');

    $project_tree.jstree({
        'types': {
            'root': {
                'icon': 'lnr lnr-briefcase'
            },
            'folder': {
                'icon': 'lnr lnr-cloud'
            },
            'file': {
                'icon': 'lnr lnr-file-empty'
            },
            'image': {
                'icon': 'lnr lnr-picture'
            },
            'home': {
                'icon': 'lnr lnr-home'
            }
        },
        'search': {
            show_only_matches: true,
            search_callback: function (str, node)
            {
                if (node.text.indexOf(str) >= 0)
                {
                    return true;
                }
            }
        },
        "plugins": [
            "types",
            "search",
            "sort"
        ],
        'sort' : function(a, b) {
            let a1 = this.get_node(a);
            let b1 = this.get_node(b);
            let isv_a1 = a1.a_attr['data-item-sort-value'];
            let isv_b1 = b1.a_attr['data-item-sort-value'];

            return (isv_a1 > isv_b1) ? 1 : -1;
        }
    });

    let showTheTree = function()
    {
        $project_tree.show();
        $project_tree_loader.hide();
    };
});

```

**Comment by Rikijs** [comment]
Hello! 
I fixed this problem by commenting option `.autoProvidejQuery` in `webpack.config.js`.
So confing now works with latest `jquery` and `jstree`!

**Comment by Rikijs** [comment]
Sorry, probably old cache issue on my part - shoud have chaked second type.
Mentioned probem still exists.

**Comment by vakata** [comment]
Seems to work fine:
http://jsfiddle.net/5cwx26qn/

Maybe there is something wrong with your stack?

**Comment by Rikijs** [comment]
Update: 
Tree works fine with `jsTree v3.3.8` and jQuery `v3.3.1`!
So the problem might be in `jQuery v3.4.1`

**Comment by vakata** [comment]
$.isFunction is deprecated in the latest copy of jsTree but that deprecation will cause trouble only after jQuery 4 is released.

**The above fiddle I provided is with jQuery 3.4.1** - so that is no issue with compatibility - once again - check your stack.

**Comment by Rikijs** [comment]
I cheked everything that i thought of.
Meanwile i created a question on StackOverflow https://stackoverflow.com/questions/56559862/jstree-v3-3-8-does-not-work-with-jquery-v3-4-1-in-symfony-4-project-with-encore.
Maybe someone has a bright idea about source of the problem.
